processor/src/main/java/org/mapstruct/ap/internal/conversion/AbstractJavaTimeToStringConversion.java

```diff
@@ -21,15 +21,15 @@
  * </p>
  * <p>
  * In general each type comes with a "parse" method to convert a string to this particular type.
- * For formatting a dedicated instance of {@link java.time.format.DateTimeFormatter} is used.
+ * For formatting a dedicated instance of {@link DateTimeFormatter} is used.
  * </p>
  * <p>
  * If no date format for mapping is specified predefined ISO* formatters from
- * {@link java.time.format.DateTimeFormatter} are used.
+ * {@link DateTimeFormatter} are used.
  * </p>
  * <p>
  * An overview of date and time types shipped with Java 8 can be found at
- * http://docs.oracle.com/javase/tutorial/datetime/iso/index.html.
+ * <a href="http://docs.oracle.com/javase/tutorial/datetime/iso/index.html">...</a>.
```

Comment:
I am not entirely sure that the 3 dots are enough. Perhaps we can some additional text here?

Reply:
In my opinion the line before the link is self explanatory and we can just have link as it is and remove the 3 dots.

Reply:
You are right @prasanth08. Let's remove tat last sentence.

Reply:
@filiphr Updated.